### DISCUSSION TITLE
what is the best config for my hardware

### DISCUSSION THREAD

**Discussion by tombicko** [swar/Swar-Chia-Plot-Manager#537] [General]
Hi all, i watched a lot of videos, a lot of text ... but i am still struggeling to understand the configuration. 

This is my pc : 

2tb NVMe FireCuda 520 SSD
intel i7-10700KF - 8 cores - 16 threads
32gb ram 3200mhz

This was my original config : 

    threads: 2
    buckets: 128
    memory_buffer: 3500
    max_concurrent: 8
    max_concurrent_with_start_early: 9
    stagger_minutes: 45
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0

First plot was created in 7,5 hours then almost all are more then 11 hours. Then i lower max_concurant to 7 and early to 8, no changes.

I am not 100% sure if threads means for all process or for 1 plot, if for one plot then why no 8 in phase 1 when each plot need 2threads - 8*2 = 16.

Please help me to adjust my config, because i am going crazy :)

Thanks a lot for help!

**Comment by babanana009** [comment]
max for phase 1: 4 
threads: 4
stagger: 25
max concurrent: 6
start early: 7
Try this config, think it’s gonna be faster. 11 h for 1 plot is way too slow for 10700k. 6-7h for 1 plot, and 6 plots per run is easy to reach for you hardware config.
A suggestion for your hardware config: use two 1T or four 0.5T ssd instead of one 2T according to my 1-mouth and 80T plotting experience.

**Reply by tombicko** [reply]
Thanks for advise, i just set it up according to your reference. Hm that is a new for me, so 2tb nvme is slower/worse then 1t nvme/0.5tb nvme ?

**Reply by babanana009** [reply]
2t nvme is definitely better than 1 or 0.5. 
But if you run two many plots on 1 ssd, the speed will drop disproportionally, which is the main reason why it takes 11h for your plot. 
For me, I have a 10850k and I use four 500g nvme and each plotting 2 plots with 25 min stagger. It takes only 6h for each plot.

**Reply by tombicko** [reply]
i see the point now...will try to get 2x1tb then instead of 2tb. Anyway i will try your setup on my pc for 24hours to see the result. Hopefully it will get better.
Tomorrow i am expecting my 2nd pc - i9 10900, with identical specs (2tb + 32gb). Think i should use the same config there.

**Reply by tombicko** [reply]
ok so after almost 24hours i see average plot is made in 9 hours(about 17-18 plots/day), it is much better but what else i can adjust to get it even faster ?

**Reply by babanana009** [reply]
1. Try changing the stagger to 50min, might be a little faster (not sure);
2. Use 2*1T nvme, definitely gonna be faster. This way you can go about 22 plots per day.

**Reply by tombicko** [reply]
1.will try it today
2. yep i am trying but it is again very difficult to find them, becoming the same thing as with GPUs
3. Is there a way to add more then i destination to the config ? or i have to create a new process for that ?
           - if i have to create new process for every new destination, for example here if i create new process should i 
              divide it by 2 ? : 
                        name 1 : 
                            max for phase 1: 4 / 2 = 2
                            threads: 4 / 2 = 2 
                            stagger: 25
                            max concurrent: 6 /2 = 3
                            start early: 7 = 4 

                        name 2: 
                             same cfg
